lib/mayaUsd/fileio/orphanedNodesManager.h

```diff
@@ -160,31 +160,28 @@ class OrphanedNodesManager : public Ufe::Observer
     // orphaned.
     bool isOrphaned(const Ufe::Path& pulledPath) const;
 
+    using PulledPrims = Ufe::Trie<PullVariantInfo>;
+    using PulledPrimNode = Ufe::TrieNode<PullVariantInfo>;
+    const PulledPrims& getPulledPrims() const { return _pulledPrims; }
+
```

Comment:
The only relevant change outside of typedef is making this function public so that the PrimUpdaterManager can call it.